core/domain/stats_jobs_one_off.py

```diff
@@ -375,22 +379,31 @@ def reduce(exp_id, values):
         stats_models.ExplorationStatsModel.save_multi(exp_stats_dicts)
 
     @classmethod
-    def _apply_state_name_changes(cls, prev_stats_dict, change_list_dict):
+    def _apply_state_name_changes(cls, prev_stats_dict, change_dicts):
         """Update the state_stats_mapping to correspond with the changes
         in change_list.
 
         Args:
             prev_stats_dict: dict. A dict representation of an
                 ExplorationStatsModel.
-            change_list_dict: list(dict). A list of all of the commit cmds from
+            change_dicts: list(dict). A list of all of the commit cmds from
                 the old_stats_model up to the next version.
 
         Returns:
             dict. A dict representation of an ExplorationStatsModel
                 with updated state_stats_mapping and version.
         """
+        RELEVANT_COMMIT_CMDS = [
```

Comment:
@seanlip Seems to look like a guard against other commands, but I think we should be switching this over to the new exp_domain.ExplorationVersionsDiff logic. No point in handling change lists explicitly. That helper domain object actually didn't exist when Joshua implemented this job, I believe.

Reply:
Not done. The problem arises with trying to interpret obsolete commit_cmds, so I don't think ExplorationVersionsDiff will help.